python/libgrass_interface_generator/ctypesgen/version.py

```diff
@@ -60,9 +59,8 @@ def compatible(v0, v1):
 def write_version_file(v=None):
     if v is None:
         v = version()
-    f = open(VERSION_FILE, "w")
-    f.write(v)
-    f.close()
+    with open(VERSION_FILE, "w") as f:
+        f.write(v)
```

Comment:
Ruff should suggest you to change it in a single Pathlib write_text call, with FURB103 rule

Reply:
@echoix I was going off of `ResourceWarnings` so I hadn't checked the Ruff outputs. I ran them just now - there were a SIM115 but no FURB103 in `ctypesgen`

Reply:
I think you need the --preview flag.

Or better idea, I had it excluded in config maybe, so maybe either give the path on the CLI, or use something like --isolated to ignore all configuration files

Reply:
@echoix Yeah I see it now. Either way updating this PR would become too messy. I'll close this PR and clone a clean copy and create separate PR for these 2 warnings. Thanks!